packages/template-retail-react-app/package.json

```diff
@@ -84,7 +89,7 @@
         },
         {
             "path": "build/vendor.js",
-            "maxSize": "270 kB"
+            "maxSize": "275 kB"
```

Comment:
Curious about this. Is this increase the result of adding `@chakra-ui/system` as a peer and dev? Is it possible that 2 versions are being included?

Reply:
Or was this the addition of the `@testing-library/dom`

Reply:
I quickly skimmed through the npm run analyze-build and I don't see any relevant change.

The bundlesize increase is minimal compared to develop. Develop was already pretty close to the threshold.

bundlesize on develop  269.65KB
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4290886347/jobs/7475393851#step:6:68

bundesize on this branch 270.43KB
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/4290886347/jobs/7475393851#step:6:68